lib/oli/delivery/evaluation/regex.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Oli.Delivery.Evaluation.Regex do
+
+  alias Oli.Delivery.Attempts.{StudentInput, Result}
+  alias Oli.Activities.Model.{Part, Response}
+
+  @behaviour Oli.Delivery.Evaluation.Evaluator
+
```

Comment:
numeric has `@impl Evaluator`, is this necessary here as well?

Reply:
added it